lib/plugins/bidirectionalData.js

```diff
@@ -289,11 +299,11 @@ function processTransformationsNGSILD(values, transformations, callback) {
  * @param {Array} values                List of reported attributes (with their name, type and value).
  * @param {Array} transformations       List of transformations to apply (with their name, type and expression).
  */
-function processTransformations(values, transformations, callback) {
-    if (config.checkNgsiLD({})){
-        processTransformationsNGSILD(values, transformations, callback)
+function processTransformations(values, typeInformation, transformations, callback) {
```

Comment:
`typeInformation` should be added as new `@param` in the function "JavaDoc"

 Note in this case the secuence is values-typeInformation-transformations-callback but in the other two functions is values-transformations-typeInformation-callback. Is this intentional?

Reply:
Fixed e0220ae and aligned the params for consistency and also aligned the calling code at line 360